aliases/available/vim.aliases.bash

```diff
@@ -5,15 +5,18 @@ VIM=$(command -v vim)
 GVIM=$(command -v gvim)
 MVIM=$(command -v mvim)
 
-[[ -n $VIM ]] && alias v=$VIM
+[[ -n $VIM ]] && alias v='$VIM'
+# open the vim help in fullscreen incorporated from
+# https://stackoverflow.com/a/4687513
+[[ -n $VIM ]] && alias vimh='${VIM} -c ":h | only"'
```

Comment:
why not do something like this:
```bash
if [[ -n $VIM ]]; then
    alias v='$VIM'
    alias vimh=  '${VIM} -c ":h | only"'
fi

Reply:
Agree - this is much cleaner and easier to read as well.